composer.lock

```diff
@@ -4994,5 +4935,5 @@
     "platform-overrides": {
         "php": "8.0"
     },
-    "plugin-api-version": "2.3.0"
+    "plugin-api-version": "2.6.0"
```

Comment:
Intended?

Reply:
This must be a side effect of *composer update*